.github/workflows/image.yml

```diff
@@ -61,14 +66,29 @@ jobs:
               .
 
       - name: push all images
-        if: github.event_name != 'pull_request'
+      #if: github.event_name != 'pull_request'
```

Comment:
Is this intentionally left in here?

Reply:
I will have to change this once the pipeline succeeds once. I need it to at least work once.